tatus

```diff
@@ -0,0 +1,126 @@
+Date:   Thu Apr 12 09:44:53 2018 -0400
+
+    Merge pull request #2125 from smartdevicelink/fix/update-apache-license
+    
+    Update Apache License
+
+[33mcommit a96ca2301e9d8380ec7e7ace098f690ff85ed75b[m
+Merge: 9daadfd e15e353
+Author: Jacob Keeler <jacob.keeler@livioradio.com>
+Date:   Wed Apr 11 16:22:09 2018 -0400
+
+    Merge pull request #2114 from smartdevicelink/hotfix/invalid_ptu_loop
+    
+    Don't force a PTU when an invalid PT is received
+
+[33mcommit 667d130e553200e2fc7ffdf222490b669b976f1c[m
+Author: JackLivio <jack@livio.io>
+Date:   Tue Apr 10 10:56:13 2018 -0400
+
+    Update Apache License
+    
+    Update license to note that the files have been modified from original form.
+    `Note: This file has been modified from its original form.`
+
+[33mcommit 9daadfd199efd06709488aa69b06f7a594bb3ea0[m
+Merge: 61fd83f 7095344
+Author: Jacob Keeler <jacob.keeler@livioradio.com>
+Date:   Mon Apr 9 15:04:55 2018 -0400
+
+    Merge pull request #2115 from smartdevicelink/hotfix/unregister_during_audio_pass_thru
+    
+    Fix/unregister while audio pass thru (#1757)
+
+[33mcommit e15e3537c9fa898b6e6142ff01e75d23391f959a[m
+Author: jacobkeeler <jacob.keeler@livioradio.com>
+Date:   Thu Apr 5 14:18:49 2018 -0400
+
+    Don't force a PTU when an invalid PT is received
+
+[33mcommit 70953440b8bca01bd63eaeba3db47e4f9fe64454[m
+Author: Sho Amano <samano@xevo.com>
+Date:   Tue Mar 27 03:35:24 2018 +0900
+
+    Fix/unregister while audio pass thru (#1757)
+    
+    * fix build error in MockMediaManager
+    * Add SetMockMediaManager
+    * add AudioPassThru start/stop tests
+    * Add new BeginAudioPassThru() and EndAudioPassThru() methods
+    * fix: don't terminate AudioPassThru when another app is unregistered
+    * Append DEPRECATED macro only in interface header file and mock file.
+    * Use @deprecated comments
+
+[33mcommit 61fd83fed0dc584019b305bebc82512d2c42b9e7[m
+Merge: 6212599 c2bdb27
+Author: Jacob Keeler <jacob.keeler@livioradio.com>
+Date:   Thu Apr 5 15:00:43 2018 -0400
+
+    Merge pull request #2005 from smartdevicelink/release/4.5.0
+    
+    Release/4.5.0
+
+[33mcommit c2bdb27e117e30752d0f223ec0e244f207883795[m
+Merge: dfbc222 a6b4aff
+Author: Jacob Keeler <jacob.keeler@livioradio.com>
+Date:   Thu Apr 5 10:13:18 2018 -0400
+
+    Merge pull request #2113 from smartdevicelink/fix/clearer_default_endpoints
+    
+    Update default endpoint urls
+
+[33mcommit a6b4aff647220112ddd99082fe623314586f1b15[m
+Author: Jacob Keeler <jacob.keeler@livioradio.com>
+Date:   Thu Apr 5 10:07:44 2018 -0400
+
+    Update default endpoint urls
+    
+     Modified default URLs to make it clearer that they should be customized by the user
+
+[33mcommit dfbc2220c44ab5cd4a1fdf29dc115579809920a0[m
+Merge: 163affd 07c274f
+Author: JackLivio <jack@livio.io>
+Date:   Wed Apr 4 17:02:24 2018 -0400
+
+    Merge pull request #2110 from smartdevicelink/fix/add_license_to_mb_files
+    
+    Fix/add license to mb files
```

Comment:
@ONyzhnyk-Luxoft remove redundant files

Reply:
fixed in [bbd5bef](https://github.com/smartdevicelink/sdl_core/pull/2669/commits/bbd5befcabccece0e86621ffcddbfa5f76eddad7)